### PR TITLE
bpo-29742: asyncio get_extra_info() throws exception

### PR DESCRIPTION
**Pull request by fafhrd91**
fix for python/asyncio#494

### CODE REVIEW

**Review by 1st1 — APPROVED** [review]
lgtm

**Comment by 1st1** [comment]
Could you please also add a NEWS entry?

**Comment by fafhrd91** [comment]
@1st1 do I need to create correspondent `bpo` issue?

**Comment by 1st1** [comment]
> @1st1 do I need to create correspondent bpo issue?

Yes, please!

**Comment by fafhrd91** [comment]
added news entry

File: Lib/asyncio/sslproto.py [hunk 1]
```diff
@@ -543,8 +543,10 @@ def eof_received(self):
     def _get_extra_info(self, name, default=None):
         if name in self._extra:
             return self._extra[name]
-        else:
+        elif self._transport is not None:
             return self._transport.get_extra_info(name, default)
+        else:
+            return None
```
Comment (1st1) [inline Lib/asyncio/sslproto.py]: `return default`, not `None`. Please also add a test for this.

Comment (fafhrd91) [reply Lib/asyncio/sslproto.py]: right. fixed

**Pull request merged**